antsibull/utils/http.py

```diff
@@ -0,0 +1,49 @@
+# Author: Felix Fontein <felix@fontein.de>
+# License: GPLv3+
+# Copyright: Ansible Project, 2020
+"""General functions for working with aiohttp."""
+
+import asyncio
+import typing as t
+
+import aiohttp
+
+
+def _format_call(command: str, args: t.Tuple[t.Any, ...], kwargs: t.Mapping[str, t.Any]) -> str:
+    arguments = [repr(a) for a in args] + ['{0}={1}'.format(k, repr(v)) for k, v in kwargs.items()]
+    return 'aio_session.{command}({args})'.format(
+        command=command,
+        args=', '.join(arguments),
+    )
+
+
+async def retry_get(aio_session: 'aiohttp.client.ClientSession',
+                    *args,
+                    acceptable_error_codes: t.Optional[t.Iterable[int]] = None,
+                    max_retries: int = 10,
+                    **kwargs):
```

Comment:
I believe you need 
```suggestion
                    **kwargs) -> None:
```

So that pyre will type check this function.